backend/app/api/v1/endpoints/cpt/cptJobs.py

```diff
@@ -102,7 +102,7 @@ async def jobs(
         "endDate": end_date.__str__(),
         "results": results_df.to_dict("records"),
         "total": jobsCount,
-        "offset": offset + size,
+        "offset": (offset + size) if size != 10000 else 0,
```

Comment:
Where does the `10000` come from here? In some of the other APIs, you *set* it if the query parameter is 0, but not here. If the call is made without `?size=<n>`, won't size be `None`?

Reply:
Yeah, without context this seems odd.

Reply:
Yeah, that's right. I missed to check for `offset` and `size` in the query parameters. Updated the code.